styles/scss/components/menu/menu-overlay.scss

```diff
@@ -0,0 +1,65 @@
+.video-react-options-close {
+  position: absolute;
+  top: 0.4em;
+  right: 0.5em;
+  color: white;
+  font-size: 2.5em;
+  z-index: 2;
+
+  &:hover {
+    cursor: pointer;
+    opacity: 0.8;
+  }
+}
+
+.video-react-options-overlay {
+  position: absolute;
+  top: 0;
+  right: 0;
+  height: 100%;
+  width: 325px;
+  background: rgba(#000000, 0.8);
+  z-index: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  font-family: "Circular", sans-serif;
+
+  span {
+    font-size: 1.5em;
+  }
+
+  .video-react-options-menu-section {
+    width: 80%;
+    align-self: center;
+
+    h3 {
+      font-size: 1.8em;
+    }
+
+    .video-react-menu-item {
+      display: flex;
+      align-items: center;
+
+      .video-react-menu-item-radio {
+        width: 1em;
+        height: 1em;
+        border-radius: 0.5em;
+        border: 1px solid white;
+        display: block;
+        margin-right: 0.5em;
+
+        &.video-react-menu-item-radio-selected, &:hover {
+          background-color: blue;
+          cursor: pointer;
+        }
+      }
+
+      p {
```

Comment:
Non-blocking feedback. I'd recommend clarity over brevity here, so something like `.video-react-menu-item-label` instead of `p` (with the corresponding change in the JSX).